var/spack/repos/builtin/packages/fastani/package.py

```diff
@@ -6,18 +6,27 @@
 from spack.package import *
 
 
-class Fastani(AutotoolsPackage):
+class Fastani(CMakePackage, AutotoolsPackage):
     """Fast, alignment-free computation of whole-genome Average Nucleotide
     Identity (ANI)"""
 
     homepage = "https://github.com/ParBLiSS/FastANI"
     url = "https://github.com/ParBLiSS/FastANI/archive/v1.33.tar.gz"
+    maintainers("snehring")
 
+    version("1.34", sha256="dc185cf29b9fa40cdcc2c83bb48150db46835e49b9b64a3dbff8bc4d0f631cb1")
     version("1.33", sha256="0b18b3074094722fb1b2247c1a1c4eb96295fff369b837f422e05072740e0013")
 
-    depends_on("autoconf", type="build")
+    build_system(
+        conditional("cmake", when="@1.34:"),
+        conditional("autotools", when="@1.33"),
```

Comment:
```suggestion
        conditional("autotools"),
```
It seems like Autotools is still supported in 1.34

Reply:
The framework's still there but the devs seem to consider cmake the primary now from what I can glean. I also can't seem to successfully test it since I run into linking errors with the zlib dep with autotools on this, which is why I checked to see if there were updates for it.

Reply:
I agree that cmake should be the primary which is why it's the default but I would still allow someone to use autotools if they want to.